index.js

```diff
@@ -0,0 +1,31 @@
+const langSec = document.querySelector(".language-section");
+const buttons = document.querySelectorAll(".button");
+const textTitle = document.querySelector(".title");
+const textDesc = document.querySelector(".description");
+
+buttons.forEach(button => {
+    button.addEventListener('click', () => {
+        langSec.querySelector(".active").classList.remove(".active");
```

Comment:
Most JS codebases (~80%) use single quotes.
```suggestion
        languageSectionEl.querySelector('.active').classList.remove('.active');
```